test/pt2_to_circle_test/test_pt2_to_circle.py

```diff
@@ -90,10 +95,19 @@ def convert_nnmodule_to_circle(
     nnmodule: torch.nn.Module,
     example_inputs: tuple,
     circle_model_path: str,
+    dynamic_shapes: dict | None = None,
 ):
     with torch.no_grad():
         _args, _kwargs = helper.get_args_kwargs(example_inputs)
-        exported_program = export(nnmodule.eval(), args=_args, kwargs=_kwargs)
+        if dynamic_shapes is not None:
+            exported_program = export(
+                nnmodule.eval(),
+                args=_args,
+                kwargs=_kwargs,
+                dynamic_shapes=dynamic_shapes,
+            )
+        else:
+            exported_program = export(nnmodule.eval(), args=_args, kwargs=_kwargs)
```

Comment:
```suggestion
        exported_program = export(nnmodule.eval(), args=_args, kwargs=_kwargs, dynamic_shapes=dynamic_shapes)
```
Is this conditional branch needed? I guess that it is fine to use like suggestion because default of `dynamic_shapes` is `None`.. 🤔
Or did you intentionally branch it?

Reply:
That's what I applied in [this commit](https://github.com/Samsung/TICO/pull/80/commits/c1c3b31d9342cc70f4c97ee34cb8c14921277037). But, this part must have slipped my mind:) Thanks.